mappings/net/minecraft/server/dedicated/AbstractPropertiesHandler.mapping

```diff
@@ -0,0 +1,31 @@
+CLASS tv net/minecraft/server/dedicated/AbstractPropertiesHandler
```

Comment:
Is this for parsing server.properties file?

Reply:
yes